lib/vagrant-openshift/action/install_openshift_base_dependencies.rb

```diff
@@ -110,6 +110,14 @@ def call(env)
 
 GOPATH=/data go get golang.org/x/tools/cmd/vet
 
+# TODO: Remove for go1.5
+# Check out a stable commit for go vet in order to version lock it to something we can work with
+pushd /data/src/golang.org/x/tools >/dev/null 2>&1
+git checkout c262de870b618eed648983aa994b03bc04641c72 
+popd >/dev/null 2>&1
+# Re-install using this version of the tool
+go install golang.org/x/tools/cmd/vet
```

Comment:
while you're in here, delete the else case for the GO_VERSION check above... code.google.com won't work any more, and that was intended for go1.3... the first branch should be used for go1.4+